docs/topics/api/overview.rst

```diff
@@ -332,6 +332,7 @@ v4 API changelog
 * 2019-07-25: added /hero/ endpoint to expose recommended addons and other content to frontend to allow customizable promos https://github.com/mozilla/addons-server/issues/11842.
 * 2019-08-01: added alias ``edition=MozillaOnline`` for ``edition=china`` in /discovery/ endpoint.
 * 2019-08-08: add support for externally hosted addons to /hero/ endpoints.  https://github.com/mozilla/addons-server/issues/11882
+* 2019-08-08: removed ``heading_text`` property from discovery api. https://github.com/mozilla/addons-server/issues/11817
```

Comment:
In `discovery.rst` there is also a mention of `heading_text` that needs to be removed.

Reply:
:see_no_evil: apparently I never committed that… fixed in 1155b51ffa